app/models/study_file_bundle.rb

```diff
@@ -18,6 +20,13 @@ class StudyFileBundle
       'BAM' => ['BAM Index'],
       'Cluster' => ['Coordinate Labels']
   }
+  # parent file type requirements by child file type
+  CHILD_REQUIREMENTS = {
```

Comment:
This is a tricky variable to name - CHILD_PARENT_REQUIREMENT is a bit long, but might better clarify that the keys are child types, and the values are a single parent type.

Reply:
Yeah I went back and forth on it for a while.  Concision seemed the better route to take.  I can update the comment for more clarity.

Reply:
7b97748